observations/marine/adt_rads_all.yaml.j2

```diff
@@ -41,7 +41,7 @@
           variables: [GeoVaLs/mesoscale_representation_error,
                       ObsError/absoluteDynamicTopography]
           coefs: [0.2,
-                  10.0]
+                  1.0]
```

Comment:
In the most recent experiments I have been using coefs: [0.2, 4.0]. Setting this to 1.0 will constrain SSH closer to ADT. Any concerns there?